### PR TITLE
 @kubernauτis [τ , ⚡] - Supporting multiple hashcat sessions in parallel using a single GPU

### PR DESCRIPTION
**Pull request by edpoptart**

### CODE REVIEW

File: neurons/Miner/pow.py [hunk 1]
```diff
@@ -186,7 +209,7 @@ def run_miner_pow(
     if len(queue) <= 0:
```
Comment (Rapiiidooo) [inline neurons/Miner/pow.py:209]: Not necessary anymore

File: neurons/Miner/pow.py [hunk 2]
```diff
@@ -135,6 +141,22 @@ def run_hashcat(
...
+                    salt=salt,
+                    mode=mode,
+                    chars=chars,
+                    mask=mask,
+                    timeout=int(timeout - execution_time),
+                    hashcat_path=hashcat_path,
+                    hashcat_workload_profile=hashcat_workload_profile,
+                    hashcat_extended_options=hashcat_extended_options,
+                    initial_start_time=start_time,
+                    session=session,
```
Comment (Rapiiidooo) [inline neurons/Miner/pow.py:158]: :clap:

**Review by nauttiilus — APPROVED** [review]
Great! Thanks for the PR.

**Comment by edpoptart** [comment]
Closing the PR since the changes were included and merged in [my other PR](https://github.com/neuralinternet/compute-subnet/pull/155)

**Pull request closed without merging**